content/events/hacktoberfest.adoc

```diff
@@ -0,0 +1,177 @@
+  You can also contribute new graphics to plugins.
+
+|=========================================================
+
+Note that this is not a full list,
+and the list will be extended depending on the interest from maintainers.
+You are welcome to contribute to any of existing Jenkins plugins...
+and even to create new ones.
+
+=== Local events
+
+Hacktoberfest is an online event,
+but there are many events being organized by open-source communities.
+You can join one of link:https://hacktoberfest.digitalocean.com/#events[these events].
+We also encourage link:/projects/jam/[Jenkins Area Meetup] organizers to
+run Jenkins-specific events in October (workshops, hackergartens, hackathons, etc.).
+
+If you are not a meetup organizer but want to host a meetup,
+you can reach out to the organizers via meetup.com resources
+(you can find a JAM link:https://www.meetup.com/pro/jenkins[here]).
+Check out the link:https://hacktoberfest.digitalocean.com/eventkit[Hacktoberfest Event Kit]
+for more info.
+
+=== FAQ
+
+You can find Hacktoberfest FAQ link:https://hacktoberfest.digitalocean.com/faq[here].
+Below you can find answer to some Jenkins-specific questions.
+
+==== Q: I am new to Jenkins, how do I start?
+
+If you are new to Jenkins,
+you could start by fixing some small and well described issues in the featured projects.
+There are lists of such newbie-friendly issues, see the links in the table above.
+You can also submit your own issue and propose a fix.
+
+==== Q: I want to work on my own plugin, is it fine?
+
+Yes, it is fine!
+Any contributions count, your role in a repository does not matter.
+Just make sure you create pull requests instead of direct pushes
+(hint: it's a best practice if you have a CI configured for your repository).
+
+==== Q: How to find documentation?
+
+Jenkins project contains lots of materials about contributing to the project.
+Here are some links which may help:
+
+* link:/participate/[Participate] - landing page for newcomer contributors
+* link:/blog/2017/08/07/intro-to-plugin-development/[Plugin Development Tutorials]
+* link:/doc/developer/[Developer Documentation]
+* link:https://gitter.im/jenkinsci/jenkins[Gitter channel] for Q&A
+
+Projects in the table above also have their own documentation to help newcomers.
+
+==== Q: How do I get reviews?
+
+All projects in the list above are monitored by their maintainers,
+and you will likely get a review within few days.
+Reviews in other repositories and plugins may take longer.
+In the case of delays, ping us in the link:https://gitter.im/jenkinsci/hacktoberfest-help[hacktoberfest-help] channel in Gitter.
+Unmerged pull-requests also count in Hacktoberfest,
+so merge delays won't block you from getting prizes.
+
+==== Q: I am stuck. How do I get help?
+
+* For non-technical questions (process and general direction) use our link:https://gitter.im/jenkinsci/hacktoberfest-[Gitter channel]
+* For technical questions please use the link:/chat[IRC chat],
```

Comment:
Since the chat page now includes Gitter, I'd leave it as "chat" rather than "IRC chat"
```suggestion
* For technical questions please use the link:/chat[chat],
```

Reply:
Will rewrite it later